### ISSUE TITLE
Wrong EDUU/EDYY Borders

### ISSUE THREAD

**Issue by lucianoasmus** [vatsimnetwork/vatspy-data-project#261]
Unfortunately the Borders for Rhein Radar and Maastricht Radar in Germany are defined wrong. The eastern part of Bremen Radar belongs to Rhein Radar.

**Comment by NelisV** [comment]
@Lotus3xiG3 Is the approved account for sector changes on vatspy. Could you weigh in here? What we have now is a compromise aligned with VATGER staff.

**Comment by Lotus3xiG3** [comment]
@lucianoasmus @NelisV Yes, we are aware of the issue : ). However, due to personal reasons I am unfortunately not longer able to function as the NAV Chief Germany and we are in the process of finding a new candidate. I will keep you posted...

**Comment by dsolesvik** [comment]
Looks good to me now
![image](https://user-images.githubusercontent.com/56816890/131520688-158579a1-f0e4-467a-bc07-146f9aed98b4.png)
![image](https://user-images.githubusercontent.com/56816890/131520730-ffd6ba8b-81d5-4c0f-96fc-f992be275f85.png)
